docs/how_to_guides/how_to_create_surrogate_models.rst

```diff
@@ -0,0 +1,168 @@
+How to create a surrogate model
+-------------------------------
+
+Surrogate modeling toolboxes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* `PySMO <https://idaes-pse.readthedocs.io/en/1.5.1/surrogate/pysmo/index.html>`_
+  
+  PySMO offers sevel surrogate modeling techniques, including:
+
+  * `Polynomial Functions <https://idaes-pse.readthedocs.io/en/1.5.1/surrogate/pysmo/pysmo_polyregression.html>`_
+  * `Kriging Functions <https://idaes-pse.readthedocs.io/en/1.5.1/surrogate/pysmo/pysmo_kriging.html>`_
+  * `Radial Basis Functions (RBF) <https://idaes-pse.readthedocs.io/en/1.5.1/surrogate/pysmo/pysmo_radialbasisfunctions.html>`_
+
+* `ALAMO <https://idaes-pse.readthedocs.io/en/1.5.1/surrogate/alamopy.html>`_
+  
+  Currently REFLO models utilize the PySMO toolbox due to its ease of use. Radial basis functions have been found to be reliable for REFLO models. The following example will demonstrate the creation of Radial Basis Function surrogate models.
+
+
+Example 1: Surrogate model for PV energy generation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Import required libraries
+
+.. testcode::
+
+  import os
+  import sys
+  import time
+  import pandas as pd
+  from pyomo.environ import Var, Constraint, units as pyunits, value, Param
+  from idaes.core.solvers import get_solver
+  from idaes.core.surrogate.sampling.data_utils import split_training_validation
+  from idaes.core.surrogate.pysmo_surrogate import PysmoRBFTrainer, PysmoSurrogate
+  from idaes.core.surrogate.surrogate_block import SurrogateBlock
+
+Define the surrogate model class and required parameters. The model class should contain the following:
+
+* build method that defines the surrogate inputs, outputs, and their respective bounds
+* get_training_validation method that loads the training and validation data
```

Comment:
I am a big fan of formatting any text that is referencing methods, classes, any code really  ``with_this_formatting`` (I am not sure what it is called). To me it stands out more and indicates this is something you could be using/looking for in the code itself. This isn't like a WaterTAP standard or anything so I won't enforce it, just a comment.

Reply:
Yeah that looks better and I figured out how to do it ``double backticks``